README.md

```diff
@@ -38,15 +38,17 @@ Deploy the "console" app to the "staging-uat" ECS cluster.
 jake deploy:app['staging-uat','console']
 ```
 
-## How to Deploy the Shipping Platform to Staging
-1. create a staging branch (Example: `staging/name-of-current-sprint`)
-2. merge in branch `GCT-314-get-shipping-working` (this branch is not in prod)
-3. Deploy shipping to staging-uat3
+#### Deploying All API Services Together
+Instead of deploying `consoleapi`, `web-api`, `worker`, and `scheduler` individually, you can instead use the following
+with the desired cluster name:
 ```bash
-jake deploy:app['staging-uat3','shipping-api']

```

Comment:
Have to checked with SRE team to confirm that this method is outdated?

Reply:
Does the SRE team have a different method for deploying shipping to staging then the standard method? If it's different then how we deploy other repos, we should update the README with their new method.

Reply:
@bearfact, @tom-con, @SealedSaint - can you answer @BriannaGreene's question about deploying `shipping-platform`?  This PR includes an update to the README that (in part) aims to remove deprecated instructions for `shipping-platform`.